### PR TITLE
chore(website): add links to Discord

### LINKED ISSUE #6628: Docs: Mention the Discord server
### Before You File a Documentation Request Please Confirm You Have Done The Following...

- [X] I have looked for existing [open or closed documentation requests](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aissue+label%3Adocumentation) that match my proposal.
- [X] I have [read the FAQ](https://typescript-eslint.io/linting/troubleshooting) and my problem is not listed.

### Suggested Changes

We're trying out having a Discord server. Previously we haven't had the funding & time to run one, but now that I'm on full time I'd like to try it out. https://typescript-eslint.io/blog/announcing-typescript-eslint-v6-beta#trying-out-v6 links to it.

### Affected URL(s)

At least...

* https://typescript-eslint.io/contributing/discussions
* https://typescript-eslint.io/contributing/issues#questions-and-support-requests

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
## PR Checklist

- [x] Addresses an existing open issue: fixes #6628
- [x] That issue was marked as [accepting prs](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aopen+is%3Aissue+label%3A%22accepting+prs%22)
- [x] Steps in [Contributing](https://typescript-eslint.io/contributing) were taken

## Overview

Adds a non-expiring infinite use link to the public server to our footer and relevant docs.

Also unifies some of the footer CSS.

### CODE REVIEW

**Comment by armano2** [comment]
image
link to github seem to be broken in header

File: packages/website/src/css/custom.css [hunk 1]
```diff
@@ -71,81 +71,44 @@ html[data-theme='dark']:root {
   --gray-border-shadow: var(--ifm-color-secondary-dark);
 }
 
-.bug-report-link {
+.image-link {
   display: flex;
   flex-direction: row;
   align-items: center;
   justify-content: start;
```
Comment (armano2) [inline packages/website/src/css/custom.css:78]:
```suggestion
  justify-content: flex-start;
```

File: packages/website/static/img/discord.svg [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<svg viewBox="0 0 127.14 96.36" xmlns="http://www.w3.org/2000/svg">
+  <path
+    fill="#000000"
+    d="M107.7,8.07A105.15,105.15,0,0,0,81.47,0a72.06,72.06,0,0,0-3.36,6.83A97.68,97.68,0,0,0,49,6.83,72.37,72.37,0,0,0,45.64,0,105.89,105.89,0,0,0,19.39,8.09C2.79,32.65-1.71,56.6.54,80.21h0A105.73,105.73,0,0,0,32.71,96.36,77.7,77.7,0,0,0,39.6,85.25a68.42,68.42,0,0,1-10.85-5.18c.91-.66,1.8-1.34,2.66-2a75.57,75.57,0,0,0,64.32,0c.87.71,1.76,1.39,2.66,2a68.68,68.68,0,0,1-10.87,5.19,77,77,0,0,0,6.89,11.1A105.25,105.25,0,0,0,126.6,80.22h0C129.24,52.84,122.09,29.11,107.7,8.07ZM42.45,65.69C36.18,65.69,31,60,31,53s5-12.74,11.43-12.74S54,46,53.89,53,48.84,65.69,42.45,65.69Zm42.24,0C78.41,65.69,73.25,60,73.25,53s5-12.74,11.44-12.74S96.23,46,96.12,53,91.08,65.69,84.69,65.69Z"
+  />
+</svg>
```
Comment (armano2) [inline packages/website/static/img/discord.svg:6]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" preserveAspectRatio="xMidYMid" viewBox="0 -29 256 256">
  <path
    fill="#000000"
    d="M217 17c-17-8-34-14-53-17l-7 14c-19-3-39-3-58 0L92 0C73 3 56 9 39 17 6 67-3 116 1 165c22 17 44 27 65 33l14-23-22-10 5-5c42 20 88 20 130 0l5 5-22 10 14 23c21-6 43-16 65-33 5-56-9-105-38-148ZM85 135c-12 0-23-12-23-26s11-26 23-26c13 0 24 12 23 26 1 14-10 26-23 26Zm86 0c-13 0-23-12-23-26s10-26 23-26c12 0 23 12 23 26s-11 26-23 26Z"
  />
</svg>
```

Comment (armano2) [reply packages/website/static/img/discord.svg]: you also want to add 

```css
    background-position: center center;
```

File: docs/contributing/Issues.mdx [hunk 3]
```diff
@@ -28,10 +28,11 @@ Please don't:
...
 ## Questions and Support Requests
 
-We do not have the bandwidth to handle questions or support requests in the issue tracker.
+The issue tracker is not an appropriate place for questions or support requests.
 You can instead:
 
+- Ask in our [Discord channel](https://discord.gg/FSxKq8Tdyg)'s `#help` server
 - Ask a question on [StackOverflow](https://stackoverflow.com/questions/tagged/typescript-eslint 'StackOverflow questions tagged with typescript-eslint') using the `typescript-eslint` tag
-- Publicly tweet [@tseslint on Twitter](https://twitter.com/tseslint)
+- Publicly toot [@tseslint on Mastodon](https://fosstodon.org/@tseslint) tweet [@tseslint on Twitter](https://twitter.com/tseslint)
```
Comment (JoshuaKGoldberg) [inline docs/contributing/Issues.mdx:36]:
```suggestion
- Publicly toot [@tseslint on Mastodon](https://fosstodon.org/@tseslint) or tweet [@tseslint on Twitter](https://twitter.com/tseslint)
```

File: packages/website/src/css/custom.css [hunk 4]
```diff
@@ -71,81 +71,44 @@ html[data-theme='dark']:root {
...
+.bug-report-link:before {
   background: url('/img/bug-report.svg') no-repeat;
 }
-.github-link {
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-  justify-content: start;
+.discord-link:before {
+  background: url('/img/discord.svg') no-repeat;
```
Comment (armano2) [inline packages/website/src/css/custom.css:95]:
```suggestion
  background: url('/img/discord.svg') no-repeat;
  background-position: center;
```

this is a nit, as this icon is not squared, we have to align it

**Review by armano2 — DISMISSED** [review]

**Comment by armano2** [comment]
ohh  i havn't noticed but looks like there is few more icons that are not showing now

image

**Pull request merged**